wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -41,7 +43,7 @@ def dev_setup
       @spec_path = 'mod/'
       @spec_helper_path = './spec/spec_helper'
       @simplecov_config = "card_simplecov_filters"
-      @gem_path = options['gem-path'] || ask("Enter the path to your local wagn gem installation: ")
+      @gem_path = ask("Enter the path to your local wagn gem installation: ") unless @gem_path
```

Comment:
The problem was it doesn't use the content of the ENV variable it uses the string "ENV['WAGN_DEV_GEM_PATH']" as path. So it doesn't ask even when the ENV variable is not set.